server/src/controllers/users.js

```diff
@@ -1,17 +1,35 @@
 const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
 const { getUserByName, createNewUser } = require('../models/users');
 
 exports.authenticateUser = (req, res) => {
   const { username, password } = req.body;
 
   getUserByName(username, (err, users) => {
-    if (err) res.json(err.message);
-    else if (user.length <= 0) {
+    if (err) console.log(err.message);
```

Comment:
` if (err) console.log(err.message);`
NO, this error handling is not acceptable, it'll break your code